frontend/src/components/ChatInterface.css

```
@@ -1,683 +1,912 @@
-  padding: var(--space-xs) var(--space-sm);
-  border-radius: var(--radius-sm);
+/* ── Message groups ──────────────────────────────────────────────────────── */
+.message-group {
+  margin-bottom: 32px;
+  animation: messageSlideIn var(--dur-deliberate) var(--ease-out);
 }
 
-.web-search-toggle:hover {
-  background: var(--color-ivory);
-  color: var(--color-text-primary);
+@keyframes messageSlideIn {
```

Stylelint (17.13.0)

[error] Expected keyframe name "messageSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)